Content.Shared/Chemistry/ReagentPrototype.cs

```diff
@@ -33,7 +33,15 @@ public void LoadFrom(YamlMappingNode mapping)
             serializer.DataField(ref _name, "name", string.Empty);
             serializer.DataField(ref _description, "desc", string.Empty);
             serializer.DataField(ref _substanceColor, "color", Color.White);
-            serializer.DataField(ref _metabolism, "metabolism", new List<IMetabolizable>{new DefaultMetabolizable()});
+
+            if (System.Reflection.Assembly.GetEntryAssembly()?.GetName().Name == "Robust.Server")
```

Comment:
This really isn't a good way to check this. It won't, for example, work in integration tests.

I'd rather you make some kind of IoC-based thing to store this.

Reply:
Done.